src/Microsoft.AspNetCore.Sockets.Client.Http/LongPollingTransport.cs

```diff
@@ -34,20 +36,20 @@ public LongPollingTransport(HttpClient httpClient, ILoggerFactory loggerFactory)
             _logger = (loggerFactory ?? NullLoggerFactory.Instance).CreateLogger<LongPollingTransport>();
         }
 
-        public Task StartAsync(Uri url, Channel<byte[], SendMessage> application)
+        public Task StartAsync(Uri url, Channel<byte[], SendMessage> application, string connectionId)
         {
-            _logger.LogInformation("Starting {0}", nameof(LongPollingTransport));
+            _connectionId = connectionId;
+            _logger.StartTransport(_connectionId);
```

Comment:
It seemed silly to include the transport in the string when the category is the transport